query-node/start.sh

```diff
@@ -20,11 +20,11 @@ docker-compose up -d db
 # Make sure we use dev config for db migrations (prevents "Cannot create database..." and some other errors)
 yarn workspace query-node config:dev
 
-# Migrate the databases
+# Prepare & migrate the databases
 yarn workspace query-node-root db:prepare
-yarn workspace query-node-root db:migrate
+yarn workspace query-node-root db:migrate # FIXME
```

Comment:
```suggestion
yarn workspace query-node-root db:migrate
```

This should be fixed by recent changes, right?

Reply:
Done in https://github.com/Joystream/joystream/pull/2854/commits/0c27ce4d5323d84180b47830f639aadd3defe9f7